File: db4ounit.extensions/src/db4ounit/extensions/tests/MultipleDb4oTestCase.java
/* Copyright (C) 2004 - 2006  Versant Inc.  http://www.db4o.com */

package db4ounit.extensions.tests;

import com.db4o.config.*;

import db4ounit.*;
import db4ounit.extensions.*;

public class MultipleDb4oTestCase extends AbstractDb4oTestCase {
	private static int configureCalls=0;
	
	public static void resetConfigureCalls() {
		configureCalls=0;
	}
	
	public static int configureCalls() {
		return configureCalls;
	}

	protected void configure(Configuration config) {
		configureCalls++;
	}
	
	public void testFirst() {
		Assert.fail();
	}

	public void testSecond() {
		Assert.fail();
	}
}

File: db4o.net/Db4oUnit.Extensions/Db4oUnit.Extensions/Tests/MultipleDb4oTestCase.cs
/* Copyright (C) 2004 - 2011  Versant Inc.  http://www.db4o.com */

using Db4oUnit;
using Db4oUnit.Extensions;
using Db4objects.Db4o.Config;

namespace Db4oUnit.Extensions.Tests
{
	public class MultipleDb4oTestCase : AbstractDb4oTestCase
	{
		private static int configureCalls = 0;

		public static void ResetConfigureCalls()
		{
			configureCalls = 0;
		}

		public static int ConfigureCalls()
		{
			return configureCalls;
		}

		protected override void Configure(IConfiguration config)
		{
			configureCalls++;
		}

		public virtual void TestFirst()
		{
			Assert.Fail();
		}

		public virtual void TestSecond()
		{
			Assert.Fail();
		}
	}
}
